### PR TITLE
[Dragon Riders] New Sheet

### PR DESCRIPTION
**Pull request by Velczer**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

- The name of this game is: < Smoczy Jeźdźcy >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: < Lans Macabre Publishing  > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: < Year Zero Engine  > _(i.e. Dungeons & Dragons, FATE)_

- [x] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

- [x] This sheet has been made on behalf of, or by, the game's publisher.
- [x] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: < 
https://wspieram.to/smoczyjezdzcy?fbclid=IwAR1EfLQf1fGaRcPdViH5oHt8xS162ADkwpaoNXxzuimssN7ns7PQmATdjDU
https://preview.drivethrurpg.com/en/product/433695/Smoczy-Jezdzcy--podrecznik-glowny?fbclid=IwAR3p5MKKxg0tcL3SkophNwcxx1JmmG4WytDQvensbPDln7RdsqbtH7HpIWM  >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

### CODE REVIEW

**Comment by NorWhal** [comment]
Sorry to do this to you @Velczer, but using `<table>` for layout purposes is not allowed, per our sheet standards. I'm going to go through and point out where things need to be changed to use a grid or flex box.

File: Dragon_Riders/dragonriders.html [hunk 1]
```diff
@@ -0,0 +1,1214 @@
...
+                                <option value="magicarkana" data-i18n="magicarkana">Arkana Magii</option>
+                                <option value="perspicacity" data-i18n="perspicacity">Przenikliwość</option>
+                                <option value="understanding" data-i18n="understanding">Rozumienie</option>
+                                <option value="history" data-i18n="history">Historia</option>
+                                <option value="inspiration" data-i18n="inspiration">Inspiracja</option>
+                                <option value="persuasion" data-i18n="persuasion">Przekonywanie</option>
+                            </select>
+                        </td>
+                    </tr>
+                </table>
```
Comment (NorWhal) [inline Dragon_Riders/dragonriders.html:100]: This should be a grid or flexbox

File: Dragon_Riders/dragonriders.html [hunk 2]
```diff
@@ -0,0 +1,1214 @@
...
+                            <td colspan="3">
+                                <label class="checkbox-label small-checkbox" style="margin:0">
+                                    <input type="hidden" name="attr_with_momentum_points" value="0">
+                                    <input type="checkbox" name="attr_with_momentum_points" value="1">
+                                </label>
+                                <span data-i18n="use-momentum-points-to-add-d6" style="font-size: 14px;">Użyj rozmachu aby dodać 6 kostek do rzutu</span>
+                            </td>
+                        </tr>
+                        
+                    </table>
```
Comment (NorWhal) [inline Dragon_Riders/dragonriders.html:406]: And here

File: Dragon_Riders/dragonriders.html [hunk 3]
```diff
@@ -0,0 +1,1214 @@
...
+                            <span data-i18n="type-of-dice">Rodzaj kości:</span>
+                            <select name="attr_magic_item_dice" style="width: 56px;padding: 0 2px;margin: 0 2px;">
+                                <option value="d10" data-i18n="d10" selected>K10</option>
+                                <option value="d12" data-i18n="d12">K12</option>
+                            </select>
+                        </div>
+                       
+                    </td>
+                </tr>
+            </table>
```
Comment (NorWhal) [inline Dragon_Riders/dragonriders.html:480]: And here

File: Dragon_Riders/dragonriders.html [hunk 4]
```diff
@@ -0,0 +1,1214 @@
...
+                            <label class="checkbox-label small-checkbox" style="margin:0">
+                                <input type="hidden" name="attr_with_dragon_power" value="0">
+                                <input type="checkbox" name="attr_with_dragon_power" value="1">
+                            </label>
+                            <span data-i18n="throw-with-dragon-power">Rzucam z mocą smoka</span>
+                        </td>
+                    </tr>
+                </tbody>
+                
+            </table>
```
Comment (NorWhal) [inline Dragon_Riders/dragonriders.html:609]: here too

File: Dragon_Riders/dragonriders.html [hunk 5]
```diff
@@ -0,0 +1,1214 @@
...
+                        <td data-i18n="dead">Martwy</td>
+                        <td>
+                            <label class="checkbox-label">
+                                <input type="checkbox" name="attr_dragon_dead" value="1">
+                                <span>M</span>
+                            </label>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
```
Comment (NorWhal) [inline Dragon_Riders/dragonriders.html:727]: Here too

**Review by NorWhal — CHANGES_REQUESTED** [review]

**Comment by Velczer** [comment]
Hello @NorWhal 
Thanks for info I changed all tables.

**Comment by nmbradley** [comment]
Hi @Velczer 

Thank you for making the requested changes. 

You may encounter an issue with this sheet with regards to your font choice. If the sheet or rolltemplates end up unstyled after merge, you may need to replace the font request in the css to use a Google font instead.

**Comment by Velczer** [comment]
@nmbradley 
Hello,
Hmm let's see I've already submitted some sheets with custom fonts loaded via files and they don't have any issues.
Fingers crossed for this to be worked again as this font is officialy created for this RPG :)

**Review by NorWhal — APPROVED** [review]

**Pull request merged**